### PR TITLE
Forprs

### PR DESCRIPTION
**Pull request by OlgaKuksa**
very simple basic structure to start

### CODE REVIEW

File: .angular-cli.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
       "testTsconfig": "tsconfig.spec.json",
       "prefix": "app",
       "styles": [
-        "styles.css"
+		"styles.css"
```
Comment (menshakou) [inline .angular-cli.json:22]: use scss `ng set defaults.styleExt scss`

File: src/app/classes/message.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export class Message {
```
Comment (menshakou) [inline src/app/classes/message.ts:1]: create folder in root dir `app/@types/index.ts` and rewrite class to interface

File: src/app/mail/mail.component.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <th scope="col">Message</th>
+      <th scope="col">Sent date</th>
+    </thead>
+    <tbody>
+      <tr *ngFor="let message of messages">
+        <td class="text-center">
+          <input type="checkbox" class="form-check-input" /> </td>
+        <td class="truncatedText">{{message.headers[0].name}}</td>
+        <td class="truncatedText">{{message.raw}}</td>
+        <td>{{message.internalDate.toLocaleString()}}</td>
```
Comment (menshakou) [inline src/app/mail/mail.component.html]: use `DatePipe` see example https://angular.io/api/common/DatePipe

File: src/app/mail/mail.component.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import { MailService } from "../services/mail.service";
+import { Message } from "../classes/message";
+
+@Component({
+  selector: "app-mail",
+  templateUrl: "./mail.component.html",
+  styleUrls: ["./mail.component.css"]
+})
+export class MailComponent implements OnInit {
+  constructor(private mailService: MailService) {}
```
Comment (menshakou) [inline src/app/mail/mail.component.ts:11]:
```
public messages: Message[];
constructor(private mailService: MailService) {}

ngOnInit() {
  this.getMessages();
}

public getMessages(): void {
  this.messages = this.mailService.getMessages();
}
```

File: src/app/services/mail.service.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { Injectable } from "@angular/core";
+import { Message } from "../classes/message";
+import { mockMessages } from "./mockMessages";
+
+@Injectable()
+export class MailService {
+  constructor() {}
+
+  getMessages() {
```
Comment (menshakou) [inline src/app/services/mail.service.ts]: add access modifier & type of return value

**Review by hankypanky666 — COMMENTED** [review]
good

**Pull request merged**